### PR TITLE
:ambulance:

### PR DESCRIPTION
**Pull request by trojikman**
1) sync.project models should be incative by default
2) Export XML: add missing eval_context field

### CODE REVIEW

File: sync/models/sync_project.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ class SyncProject(models.Model):
     name = fields.Char(
         "Name", help="e.g. Legacy Migration or eCommerce Synchronization", required=True
     )
-    active = fields.Boolean(default=True)
+    active = fields.Boolean(default=False)
```
Comment (trojikman) [inline sync/models/sync_project.py]: @yelizariev What do you think about it?

Comment (yelizariev) [reply sync/models/sync_project.py]: I't good for module creating, but not for a custom Project created via UI

File: sync/wizard/sync_make_module.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ def act_makefile(self):
         root = etree.Element("odoo")
         project = self.project_id.with_context(active_test=False)
         records = [
-            (project, ("name", "active", "common_code")),
+            (project, ("name", "active", "eval_context", "common_code")),
```
Comment (yelizariev) [inline sync/wizard/sync_make_module.py:84]: Use my commits from v12 please
https://github.com/itpp-labs/sync-addons/commit/dbcd038791eb5b9798dfdb388dd99c7bc24c25ce

Comment (yelizariev) [reply sync/wizard/sync_make_module.py:84]: Also this https://github.com/itpp-labs/sync-addons/pull/166/commits/41dc79724e422b1ce733df2271b129fdb46a38a3

**Comment by trojikman** [comment]
I approve to merge it now

**Pull request merged**